.circleci/config.yml

```diff
@@ -9,6 +9,26 @@ references:
     rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "content/Gemfile.lock" }}
 
 jobs:
+  website-test:
+    machine:
```

Comment:
This probably doesn't need a machine image and could use the docker container but it looks like in Travis it used 2.3.1 and that circleci/ruby version doesn't exist. Since the build for the deploy uses the `*MIDDLEMAN_IMAGE` I would just reuse that here.

Reply:
I'm actually using the machine executor here because the `website-test` target mounts volumes in the docker run cmd, and I couldn't find a nice way to do that from a base docker image: https://circleci.com/docs/2.0/building-docker-images/#mounting-folders. At the bottom of the docs here, it suggests using the machine executor to avoid the mess :D